skimage/_shared/testing.py

```diff
@@ -18,39 +20,30 @@
 SKIP_RE = re.compile("(\s*>>>.*?)(\s*)#\s*skip\s+if\s+(.*)$")
 
 
-def _assert_less(a, b, msg=None):
+def assert_less(a, b, msg=None):
```

Comment:
Forget about this as well. I've created a separate issue - https://github.com/scikit-image/scikit-image/issues/2610.